### ISSUE TITLE
Support by Fire Point Draw Rules and Example

### ISSUE THREAD

**Issue by joebayles** [Esri/joint-military-symbology-xml#337]
MIL-STD-2525 alludes to variation in the angle of the two lines for the arrows created by points three and four. However, this variation is seldom used in digital representations, presumably because of the difficulty in rendering it and the fact that the message format doesn't support it. For instance, here it is used in a doctrinal publication:
![image](https://cloud.githubusercontent.com/assets/5016557/26333732/ddb7b5dc-3f14-11e7-8133-2fef470f6ad1.png)

_Note that the lines showing the field of fire don't necessarily match the angle of the support by fire fields_

So, my suggestion is to set this as a line feature with two points, and fix the brackets and arrows on either side (remove PT 3 and PT 4).

Also, I would update the example to include a friendly unit icon or remove the enemy unit icon.

For reference:
![image](https://cloud.githubusercontent.com/assets/5016557/26333796/61f7c530-3f15-11e7-92d7-03b9ee0813b7.png)

**Comment by Dbarnes1** [comment]
@joebayles I think that this symbol should actually be a multi-layer marker symbol with two attribute driven symbol components/layers. How someone would draw the symbol would work like this:

1. User draws the symbol by clicking on the map at the centerpoint of the support by fire
2. User can rotate and scale the symbol depending on the size
3. User sets length of lateral support by fire line (attribute driven)
4. User sets angle of two fires arrows (attribute driven)

I don't think this is currently implementable in ArcGIS, but is definitely in the realm of possibility with symbol overrides, etc. Something to bring to both the SSMC and core.

**Comment by joebayles** [comment]
@Dbarnes1 agreed, I guess I didn't really make my committee suggestion clear. Schema has to be updated.

**Comment by Dbarnes1** [comment]
@joebayles ideally these special case symbols can have UI where user enters angle/line length that isn't locked into our schema (Like how you can apply extrusion without it being in your schema), but we're definitely on the same page.

**Comment by ottenw** [comment]
@joebayles if you are attending the next SSMC in person, this might be a good thing to discuss with the Army representative.  If the angle of the arrows is doctrinally significant, the message standards can be updated.  It takes a few years between the time a message standard is updated and the new version is implemented, tested, and fielded; in the mean time what you are implementing is very usable.

**Comment by joebayles** [comment]
@ottenw - After discussing it at the committee, two things:
Suggest we only vary the length between points 1 and 2

![picture1](https://user-images.githubusercontent.com/5016557/28531624-41d85644-704c-11e7-92c2-df0b6f6b8c6e.png)

Army rep will discuss with proponent agency.

**Comment by joebayles** [comment]
This generated quite a bit of conversation. BLUF: This is a draw rule and implementation issue that affects several control measures and tactical mission tasks. The suggestion is that Army and Marine Corps will review and generate CPs to clarify guidance.

**Comment by joebayles** [comment]
Esri should review control measures and ensure that the list is complete. Will open another issue for that.

**Comment by joebayles** [comment]
According to the US Army, the only difference from the above graphic is that the angle between the arrows and the base line should be variable, though the length of the arrows should not be. The committee needs some help in deciding what the draw rule should like like, but in this case, it will still probably have to be 4 points. This will be the same for Attack by Fire, just with the one centered arrow, instead of two.

FYI @jeconley @kerryrobinson @ralfgottschalk @lfunkhouser
